<table border="1" class="dataframe">
    <thead>
      <tr style="text-align: right;">
        <th></th>
        <th>Speaker</th>
        <th>Affiliation</th>
        <th>Title</th>
        <th>Abstract</th>
      </tr>
      <tr>
        <th>Date</th>
        <th></th>
        <th></th>
        <th></th>
        <th></th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <th>"2023-05-22"</th>
        <td>"Dr. Hua Zhou"</td>
        <td>"University of California Los Angeles"</td>
        <td>"Inferring Within-Subject Variances From Intensive Longitudinal Data"</td>
        <td>"The availability of vast amounts of longitudinal data from electronic health records (EHR) and personal wearable devices opens the door to numerous new research questions. In many studies, individual variability of a longitudinal outcome is as important as the mean. Blood pressure fluctuations, glycemic variations, and mood swings are prime examples where it is critical to identify factors that affect the within-individual variability. We propose a scalable method, within-subject variance estimator by robust regression (WiSER), for the estimation and inference of the effects of both time-varying and time-invariant predictors on within-subject variance. It is robust against the misspecification of the conditional distribution of responses or the distribution of random effects. It shows similar performance as the correctly specified likelihood methods but is 10_ ~10_ times faster. The estimation algorithm scales linearly in the total number of observations, making it applicable to massive longitudinal data sets. The effectiveness of WiSER is illustrated using the accelerometry data from the Women's Health Study and a clinical trial for longitudinal diabetes care."</td>
      </tr>
      <tr>
        <th>"2023-04-24"</th>
        <td>"Dr. Quian Li"</td>
        <td>"St. Jude Children's Research Hospital"</td>
        <td>"Statistical Frameworks for Longitudinal Metagenomic and Transcriptomic Data"</td>
        <td>"Longitudinal sampling has become popular in the omics studies, such as microbiome and transcriptome. To identify operational taxonomy units (OTUs) signaling disease onset, a powerful and cost-efficient strategy was selecting participants by matched sets and profiling their temporal metagenomes, followed by trajectory analysis. We proposed a joint model with matching and regularization (JMR) to detect OTU-specific trajectory predictive of host disease status. The between- and within-matched-sets heterogeneity in OTU relative abundance and disease risk were linked by nested random effects. The inherent negative correlation in microbiota composition was adjusted by incorporating and regularizing the top-correlated taxa as longitudinal covariate, pre-selected by Bray-Curtis distance and elastic net regression. For the longitudinal bulk transcriptomes, we propose a statistical framework ISLET to infer individual-specific and cell-type-specific transcriptome reference panels. ISLET models the repeatedly measured bulk gene expression data to optimize the usage of shared information within each subject. ISLET is the first available method to achieve individual-specific reference estimation in repeated samples. In the simulation study and an application to a large-scale metagenomic study, JMR outperformed the competing methods and identified important taxa in infantsÕ fecal samples with dynamics preceding host autoimmune status. We also show outstanding performance of ISLET in the reference estimation and downstream cell-type-specific differentially expressed genes testing in simulation. An application of ISLET to the longitudinal PBMC transcriptomes in the same study confirms the cell-type-specific gene signatures for early-life autoimmunity."</td>
      </tr>
      <tr>
        <th>"2023-03-27"</th>
        <td>"Dr. Patrik Breheny"</td>
        <td>"University of Iowa"</td>
        <td>"False Discovery Rates for Penalized Regression Models"</td>
        <td>"Penalized regression is an attractive methodology for dealing with high-dimensional data where classical likelihood approaches to modeling break down. However, its widespread adoption has been hindered by a lack of inferential tools. In particular, penalized regression is very useful for variable selection, but how confident should one be about those selections? How many of those selections would likely have occurred by chance alone? In this talk, I will review recent developments in this area, with an emphasis on my work and that of my recent graduate students."</td>
      </tr>
      <tr>
        <th>"2023-02-20"</th>
        <td>"Dr. Ccile An"</td>
        <td>"University of Wisconsin-Madison"</td>
        <td>"Methods for Phylogenetic Networks"</td>
        <td>"Phylogenetic networks and admixture graphs can represent the past history of a group of species or populations and how they diversified. Unlike trees, networks can represent events such as migration between populations, admixture, hybridization between species, or recombination between viral strains. I will give an overview of how phylogenetic networks are used and the difficulties of estimating phylogenetic networks from genome-wide data. Then I will focus on the characterization of what is (or is not) knowable about the network based on genetic distance data."</td>
      </tr>
      <tr>
        <th>"2022-10-31"</th>
        <td>"Chenhao Zhao"</td>
        <td>"Geisel School of Medicine, Dartmouth College"</td>
        <td>"MatrixLM: a flexible, interpretable framework for high-throughput data"</td>
        <td>"The Matrix Linear Model (MLM) is an efficient and computationally feasible solution to association analysis for biomedical high-throughput data. Sen and Liang (2018) developed the MatrixLM.jl package in Julia.  programming language, providing core functions to estimate matrix linear models. The project's main goal was to collaborate on user-friendly documentation, increase testing features, and improve certain coding functionalities. We used simulated data to demonstrate how to use the package and used a case study example based on an actual disease metabolomics study to showcase MLM's benefits. Nonalcoholic fatty liver disease (NAFLD) is a progressive liver disease that is strongly associated with type II diabetes 2. Using Matrix Linear Models, our analysis investigated the association between metabolite characteristics (e.g., pathways) and patient characteristics such as type II diabetes."</td>
      </tr>
    </tbody>
  </table>